content/collections/experiment_troubleshooting/en/new-experiment-run.md

```diff
@@ -9,20 +9,6 @@ updated_at: 1717179968
 ---
 Creating a new run of an existing experiment can be useful if you had instrumentation issues that affected data quality, and you've since fixed them. When you create a new run, you exclude any previous user data from the monitoring and analysis of your experiment.
 
-## Prerequisites
```

Comment:
@bgiori We don't need this table right since it appears below in the "Experiment key" section right? The minimum SDK version and using amplitude default exposure tracking is only if they are using the experiment key method and not the dates method right?

Reply:
Correct, we can delete these prerequisites